core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -147,34 +220,29 @@ var LearnerDashboardPage = function() {
     expect(await feedbackMessage.first().getText()).toMatch(message);
   };
 
-  this.checkIncompleteExplorationSection = async function(explorationTitle) {
-    await this.navigateToInCompleteSection();
-    await this.navigateToIncompleteExplorationsSection();
-    await this.expectTitleOfExplorationSummaryTileToMatch(
-      explorationTitle);
-  };
+  this.navigateToCommunityLessonsAndCheckIncompleteExplorations = (
+    async function(
+        explorationTitle) {
+      await this.navigateToCommunityLessonsSection();
+      await this.expectTitleOfExplorationSummaryTileToMatch(explorationTitle);
```

Comment:
It's unclear to me what `expectTitleOfExplorationSummaryTileToMatch` has to do with `checkIncompleteExplorations`

Reply:
We will compare the title of the `oppia-exploration-summary-tile` in the incomplete section of the community lessons tab with the incomplete exploration title.

Reply:
But where are we specifying that we check the incomplete section instead of the complete section?

Reply:
Done. PTAL!